src/commands/rename.rs

```diff
@@ -0,0 +1,109 @@
+use super::*;
+use crate::core::Target;
+use clap::Arg;
+use tracing_batteries::prelude::*;
+use crate::git;
+
+pub struct RenameCommand;
+crate::command!(RenameCommand);
+
+#[async_trait]
+impl CommandRunnable for RenameCommand {
+    fn name(&self) -> String {
+        String::from("rename")
+    }
+
+    fn app(&self) -> clap::Command {
+        clap::Command::new(self.name())
+            .about("renames a repository on your local machine")
```

Comment:
I think it'd be cool to reserve the `mv` alias for this command as well, what do you think?

Reply:
Oh, I thought I added that? Might have accidentally removed it again... :/